src/commands/player/DailyCommand.js

```diff
@@ -19,13 +19,12 @@
 	if (activeObject.nature === NATURE.NONE) {
 		if (activeObject.id !== JsonReader.models.inventories.objectId) {
 			// there is a object that do nothing in the inventory
-			sendErrorMessage(message.author, message.channel, language, JsonReader.commands.daily.getTranslation(language).objectDoNothingError);
+			return sendErrorMessage(message.author, message.channel, language, JsonReader.commands.daily.getTranslation(language).objectDoNothingError);
 		}
-		else {
+
 			// there is no object in the inventory
-			sendErrorMessage(message.author, message.channel, language, JsonReader.commands.daily.getTranslation(language).noActiveObjectdescription);
-		}
-		return;
+			return sendErrorMessage(message.author, message.channel, language, JsonReader.commands.daily.getTranslation(language).noActiveObjectdescription);
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 3.](https://app.codacy.com/gh/DraftBot-A-Discord-Adventure/DraftBot/pullRequest?prid=7559804)
```suggestion
		return sendErrorMessage(message.author, message.channel, language, JsonReader.commands.daily.getTranslation(language).noActiveObjectdescription);
```